package.json

```
@@ -17,15 +22,15 @@
     "format:prettier": "prettier \"**/*.{cjs,html,js,json,md,ts}\" --write",
     "ignore-sync": "ignore-sync .",
     "lint": "eslint \"**/*.{js,ts}\"",
-    "nuke": "rm -rf node_modules && npm install && lerna exec 'rm -rf node_modules' && lerna bootstrap && npm run clean",
-    "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean",
+    "nuke": "rm -rf node_modules && npm exec -ws -c 'rm -rf node_modules' && npm ci",
+    "upgrade": "rm -rf node_modules package-lock.json && npm exec -ws -c 'rm -rf node_modules' && npm i",
     "test": "wireit",
     "test:lit-starter-js": "wireit",
     "test:lit-starter-ts": "wireit",
     "test:windows": "wireit",
     "prepare": "husky install",
     "changeset": "changeset",
-    "version": "npm run changeset version && npm run update-version-vars && lerna bootstrap -- --package-lock-only",
+    "version": "npm run changeset version && npm run update-version-vars && npm i",
```

I added the the `lerna bootstrap` step in an effort to generate updated versions of package-lock files for the updated packages (which ended up not working). Since we no longer have sub package package-locks, this step is no longer necessary.
```suggestion
    "version": "npm run changeset version && npm run update-version-vars",
```